AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -30,6 +36,11 @@ internal final class QRCodeComponent: ActionComponent, Localizable {
     /// - Parameter style: The component UI style.
     public init(style: QRCodeComponentStyle = QRCodeComponentStyle()) {
         self.style = style
+        
+        self.timeLeft = timeoutInterval
+        self.progress = Progress()
```

Comment:
this can be initialized where the `progress` is declared

```
private let progress: Progress = Progress()
```

Reply:
good catch!